### PR TITLE
zh-trans: add includes/default-storage-class-prereqs.md

### PR DESCRIPTION
**Pull request by xichengliudui**
zh-trans: add includes/default-storage-class-prereqs.md

### CODE REVIEW

File: content/zh/includes/default-storage-class-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+您需要有一个带有默认[存储类](/docs/concepts/storage/storage-classes/)的动态持续卷供应程序，或者自己[静态的提供持续卷](/docs/user-guide/persistent-volumes/#provisioning)来满足这里使用的[持续卷请求](/docs/user-guide/persistent-volumes/#persistentvolumeclaims)。
```
Comment (AdamDang) [inline content/zh/includes/default-storage-class-prereqs.md]: 持续卷-》持久卷

Comment (xichengliudui) [reply content/zh/includes/default-storage-class-prereqs.md]: 好的

**Comment by markthink** [comment]
@xichengliudui 意见要更新一下.

**Comment by tengqm** [comment]
@xichengliudui I have copy edited the PR, did you check it?

**Pull request closed without merging**